Aptoide/aptoide-client-v8
# [ASV-284] Fixed bug where followed stores wouldn't be correctly updat…

**What does this PR do?**

This PR aims to fix a bug where followed stores wouldn't be properly filled in a listSeachApps request;

**Database changed?**

No

**Where should the reviewer start?**

- [ ] SearchManager.java

**How should this be manually tested?**

Log off the application. Use ADB on studio to clear data and restart. Open charles. Do a search -> two requests for list search apps should be there: one with an empty store_ids field and another without that field at all. Now go to the stores tab. Follow a store. Do another search -> should have two requests similar to the ones above but the store_ids field show now be updated with the store you just followed. Repeat follow store -> search steps a couple more times and verify the ids are being added to the request list;

**What are the relevant tickets?**

  Tickets related to this pull-request: [ASV-284](<https://aptoide.atlassian.net/browse/ASV-284>)
  (Bug fix for trending icon color change appended here) :
        [ASV-276](<https://aptoide.atlassian.net/browse/ASV-276>)


**Code Review Checklist**
